gdx/src/com/badlogic/gdx/utils/IntIntMap.java

```diff
@@ -533,6 +533,30 @@ private int hash3 (int h) {
 		return (h ^ h >>> hashShift) & mask;
 	}
 
+	@Override
+	public int hashCode () {
+		int h = 0;
+		for (Entry entry : entries()) {
+			h += entry.key * 31 + entry.value;
```

Comment:
> I think the value needs prime multiplication.

Doesn't that result in a worse hash function? This will result in the same hash code for a reversed map.